BasiliskII/src/Windows/posix_emu.cpp

```diff
@@ -1123,3 +1123,28 @@ int my_write( int fd, const void *buffer, unsigned int count )
 	D(bug("write(%ld,%08x,%ld) = %d\n", fd, buffer, count, result));
 	return result;
 }
+
+FILETIME getFileTime(time_t time) {

```

Comment:
Please use a consistent naming convention with the rest of the file - e.g. get_file_time().

Also, make this function static if you're not using it outside this file.